### DISCUSSION TITLE
v0.25.1 (⚠️ Influxdb breaking change)

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1894] [Releases]
This release fundamentally rewrites the integration with Influxdb and requires you to restate your history (yay new feature!) or create a new bucket for tests going forward. Make sure to read the full list of changes in #1866.

## What's Changed
* [Bug] Don't use factory to test influxdb connection by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1892
* Release v0.25.1 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1893

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.25.0...v0.25.1

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.25.1'>v0.25.1 (⚠️ Influxdb breaking change)</a>.</em>

**Comment by ShlomiD83** [comment]
Hi @alexjustesen 
Can you please explain how to restate my history?

**Reply by alexjustesen** [reply]
What I did was...

1. Delete my bucket as it only contained speedtest data.
2. Create a new bucket with the same name.
3. Click the button on the Influxdb settings page "Export current results".
4. Watch the data feed into your bucket.

There are probably other ways to do this for people more versed in Influxdb but this worked for my needs.

**Reply by ShlomiD83** [reply]
@alexjustesen thanks, did just that.
I can see the data in the bucked, now wait for Grafana to display it properly.

**Reply by ShlomiD83** [reply]
![image](https://github.com/user-attachments/assets/0350c6cb-6a2c-44b4-a15f-f7d5c0fab93a)
I can see the results listed below but the data isn't populated.

**Reply by svenvg93** [reply]
The data is there, mostly because of different tags the data is not combined. Try to add Merge series/tables to the charts should work. The refactoring changed a lot so you will need to make changes to your dashboard to work again.

**Reply by ShlomiD83** [reply]
@svenvg93 I'll try, not really sure how to do that..

**Reply by svenvg93** [reply]
When you edit a chart you need to add the transformation called "Merge series/tables"

**Reply by ShlomiD83** [reply]
> When you edit a chart you need to add the transformation called "Merge series/tables"

@svenvg93 thanks, I've managed to fix everything but the top average panel. 
It already has the transformation but no data is showing.

**Reply by svenvg93** [reply]
I see that the original builder of the dashbaord is working on getting everything adjusted after the refactoring. Might want to wait for a new version; https://github.com/masterwishx/Speedtest-Tracker-v2-InfluxDBv2/issues/8#issuecomment-2523615866

**Reply by ShlomiD83** [reply]
Thanks @svenvg93

**Comment by vdrover** [comment]
Anything to worry about if we're not using InfluxDB? Also, why use InfluxDB at all?

**Reply by alexjustesen** [reply]
> Anything to worry about if we're not using InfluxDB? 

None at all, nothing to see here.

> Also, why use InfluxDB at all?

Provides a pattern for people who create Grafana dashboards that may combine multiple data streams into one visualization.
